student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age)
```

Comment:
- [ ] The other parameters needed to create a person need to be added since the Student class inherits from Person.

- [ ] The order of the arguments in the `super` method should be followed

  ```
  def initialize(classroom, age:, name: 'Unknown', parent_permission: true)
    super(age: age, name: name, parent_permission: parent_permission)
  ```